.travis.yml

```diff
@@ -1,16 +1,15 @@
+sudo: false
 language: python
 env:
-  - REQUESTS_VERSION=2.1.0
-  - REQUESTS_VERSION=2.8.1
-  - REQUESTS_VERSION=2.9.1
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-install:
-  - if [[ $TRAVIS_PYTHON_VERSION == 2.6* ]]; then export PYTEST_ADDOPTS='-p no:pylint'; fi
-  - pip install -r requirements.txt
-  - pip install --upgrade --force-reinstall requests==$REQUESTS_VERSION
-script: py.test
+    - TOXENV=py27-requests-current
```

Comment:
What the above means in usability is that currently if a test fails in one of the environments travis will display the failed environment from the UI. e.g.: https://travis-ci.org/john-kurkowski/tldextract/builds/93708810

If no TOXENV vars are defined then tox will run all permutations under a single build, hiding the details of which environment failed deep in the logs. Tox makes it easy to see which envs failed but not as nice as travis web UI.